src/kirby/components/list/list-item/list-item.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface ListItem {
```

Comment:
The `kirby-list` should not have any knowledge of the format of the items handed to it.
So i think this is not a route we should take, as it's opens a can of worms we don't want to work with 😄 

We hand back the `item` to the user of the list in the templates. So altering the `item` is a bad idea, as we risk to overwrite data.

So we will have to rethink how a item is marked as selected in the code.

Reply:
I added this, so I had a typesafe way of ensuring that the item had an id. This is used for checking if an item is selected. Without an interface, and an id, how would you check if an item is selected?

Reply:
As talked about I will:
1. Add _id property with GUID on item input
2. Remove_id on data output
3. Delete ListItem interface